mergify_engine/rules/types.py

```diff
@@ -63,13 +63,13 @@ def _ensure_complete():
 )
 
 
-def Jinja2(value):
+def Jinja2(value, additional_variables=None):
```

Comment:
Might be cleaner to have that passed a list, where this function would just put `""` string to check the template.

Reply:
I kept the dict because the value may not be a string.